apps/site/assets/ts/helpers/service.ts

```diff
@@ -44,6 +49,11 @@ export const isCurrentValidService = (
   service: Service,
   currentDate: Date = new Date()
 ): boolean => {
+  // check against added dates
+  if (isInAddedDates(service, currentDate)) {
+    return true;
+  }
+
   // check against removed dates
```

Comment:
This is the key change. Everything else is mainly testing coverage enhancements.